examples/semantic_segmentation/semantic_segmentation/seed_test_run.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import sys
+from argparse import ArgumentParser
+from argparse import Namespace
+from typing import List
+
+from semantic_segmentation.constants import BUCKET
+from semantic_segmentation.constants import DATASET
+from semantic_segmentation.evaluator import evaluate_semantic_segmentation
+from semantic_segmentation.workflow import Inference
+from semantic_segmentation.workflow import Model
+from semantic_segmentation.workflow import SegmentationConfiguration
+from semantic_segmentation.workflow import TestSample
+from semantic_segmentation.workflow import TestSuite
+
+import kolena
+from kolena.workflow.asset import BinaryAsset
+from kolena.workflow.test_run import test
+
+
+def seed_test_run(model_name: str, test_suite_names: List[str]) -> None:
+    def infer(test_sample: TestSample) -> Inference:
+        basename = test_sample.metadata["basename"]
+        locator = f"s3://{BUCKET}/{DATASET}/results/{model_name}/{basename}_person.npy"
```

Comment:
Might want to do something to sanitize model name so it's safe for a directory name.

Reply:
@sandalns could you take a look at this please?

Reply:
Will take a look